packages/system/src/valueConverters.ts

```diff
@@ -7,3 +7,8 @@ export const spaceConverter: ValueConverter = (v) => {
   const userTheme = theme.getUserTheme();
   return userTheme.spacings?.[v] || toCssUnit(v);
 };
+
+export const sizeConverter: ValueConverter = (v) => {
+  const userTheme = theme.getUserTheme();
+  return userTheme.sizes?.[v] || toCssUnit(v);
```

Comment:
I'm not very familiar with ESLint, but did this line not throw errors like `eslint@typescript-eslint/no-unsafe-return` or `eslint@typescript-eslint/no-unsafe-member-access` in your local env? Since the CI isn't failing, I'm wondering if it's just an issue on my end?

Reply:
@poteboy No error in my machine. Maybe you missed to run `pnpm build`?

Reply:
I re-ran `pnpm build` but still encountered the error. 
Interestingly, the error disappeared when I turned off the [VSCode plugin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).